### DISCUSSION TITLE
LATEST SIMHUB APP FAILS SEEN/DETECTED BY ETS OR ATS

### DISCUSSION THREAD

**Discussion by golivezambia** [sjdawson/TruckSimulatorPlugin#5] [Q&A] [answered]
on my ATS and ETS Simhub 7.01.4 is able to be detected as an SDK, but later versions are not being detected at all. Were have i missed

**Comment by sjdawson** [comment, marked as answer]
Hi @golivezambia,

That's odd, because the later versions of SimHub shouldn't make that much of a difference. However, try following the configuration setup guide for those games again from the latest version of SimHub. The files you need to copy over are likely newer than the ones already in place.

https://github.com/SHWotever/SimHub/wiki/SimHub-Basics----Games-config-and-troubleshooting

However, I only write a plugin for SimHub here, so for further support, you'd be better directing it at the main SimHub repository (https://github.com/SHWotever/SimHub/issues), or the Discord server (https://discord.com/invite/nBBMuX7)

**Comment by Lukas1736** [comment]
you called it instant fule consumtion but i can not find anyinstant fule consumtion??? Wer ist it am i just to dum to find it or is ther non im if ther is non pleas  add it THX

**Comment by sjdawson** [comment]
Hi @Lukas1736,

My plugin makes some average fuel consumption properties available, which may also be known as instant consumption;
https://sjdawson.gitbook.io/trucksimulatorplugin/features/drivetrain#average-consumption-l-100mi-mpg-uk-mpg-us

These are all calculated from the base property available from the game's direct telemetry: `DataCorePlugin.GameRawData.TruckValues.CurrentValues.DashboardValues.FuelValue.AverageConsumption`

If you need further help with this - please open a new discussion: https://github.com/sjdawson/TruckSimulatorPlugin/discussions/new?category=q-a
